### DISCUSSION TITLE
How frequently do MQTT values need to be published to prevent dropping from loop packet?

### DISCUSSION THREAD

**Discussion by kevinkey619** [bellrichm/WeeWX-MQTTSubscribe#159] [General]
My pressure readings keep randomly getting dropped from the loop packet. How frequently do the MQTT values need to be published to prevent this from happening?

**Comment by bellrichm** [comment]
Lets work through your problems in one place, https://github.com/bellrichm/WeeWX-MQTTSubscribe/discussions/157
